### DISCUSSION TITLE
Can a remote KeePassXC access local YubiKey?

### DISCUSSION THREAD

**Discussion by Programmer-Magnus** [keepassxreboot/keepassxc#6559] [Q&A] [answered]
I work at home on a local Windows PC and I use Windows Remote Desktop to access my remote work Windows PC.
Is it possible to have the YubiKey in my local PC and let KeePassXC open a database on my remote PC using this YubiKey?
Both KeePassXC and database are on the remote PC.

**Comment by droidmonkey** [comment, marked as answer]
See https://github.com/keepassxreboot/keepassxc/issues/6439

**Reply by Programmer-Magnus** [reply]
Thank you. I will try that solution.
Cheers
